### DISCUSSION TITLE
Cannot place all ranks on node list

### DISCUSSION THREAD

**Discussion by RuiyuSun** [NOAA-EMC/global-workflow#2896] [Archive] [answered]
I am trying to conduct a high resolution (C1152) coupled experiment on dogwood. Because the forecast length is fairly long the default number of nodes (tasks) can't complete the gfsfcst job. The default layouts are 
layout_gfs_x=8 
layout_gfs_y=16. 

I tried to increase the number of nodes by increasing the layouts. An example is the following pair. 
layout_gfs_x=12
layout_gfs_y=24

But I had the following error. 
**Cannot place all ranks on node list**

 mpiexec -l -n 10240 -ppn 128 --cpu-bind depth --depth 1 /lfs/h2/emc/stmp/ruiyu.sun/RUNDIRS/HR43/gfs.2020012600/gfsfcst.2020012600/fcst.28576/ufs_model.

Any suggestion? 

Thanks,
 Ruiyu

**Comment by jiandewang** [comment]
@RuiyuSun 
I cloned latset g-w and tried C1152 S2SW, it works fine for 24x16 and 12x24 layout. See my log file on dogwood
12x24: /lfs/h2/emc/ptmp/jiande.wang/HR4-work/HR4-20191212/COMROOT/2019121200/HR4-20191212/logs/2019121200
24x16: /lfs/h2/emc/ptmp/jiande.wang/HR4-work/HR4-20191203/COMROOT/2019120300/HR4-20191203/logs/2019120300

**Comment by jiandewang** [comment, marked as answer]
the g-w I cloned:
commit 49f697a4ac3d32a66d6108ebb3e07a9d9307493e (HEAD -> develop, origin/develop, origin/HEAD)
Author: DavidNew-NOAA <134300700+DavidNew-NOAA@users.noreply.github.com>
Date:   Sat Sep 7

**Comment by jiandewang** [comment]
my g-w
 /lfs/h2/emc/couple/noscrub/Jiande.Wang/HR4-test/global-workflow

**Reply by RuiyuSun** [reply]
Thanks! @jiandewang I copied your experiment directory. It worked fine for 12X24 layout.

**Reply by RuiyuSun** [reply]
I also used your global workflow experiment.
